go/samples/menu/s02.go

```diff
@@ -50,9 +50,7 @@ func setup02(g *genkit.Genkit, m ai.Model) error {
 
 	genkit.DefineFlow(g, "s02_menuQuestion",
 		func(ctx context.Context, input *menuQuestionInput) (*answerOutput, error) {
-			resp, err := dataMenuPrompt.Generate(ctx, g,
-				dotprompt.WithInput(input),
-			)
+			resp, err := dataMenuPrompt.Generate(ctx, g, dotprompt.WithInput(input))
```

Comment:
This flow is generating the following error on tool use:
`2025/02/27 19:55:52 ERROR request end reqID=11 err="exceeded maximum tool call iterations (0)"`